Assets/Source/Components/Player/PlayerBehavior.cs

```diff
@@ -138,7 +140,7 @@ private void UpdateExternalVelocity()
 
         }
 
-        public void ReactToProjectileHit(Collision2D collision, int baseDamage)
+        public void ReactToHit(Collision2D collision, int baseDamage, float partDamageMultiplier)
```

Comment:
will this partDamageMultiplier be used eventually like once every part of the ship is defined by its own collider?

Reply:
yusssss!